scripts/make_wheel.sh

```diff
@@ -45,7 +45,7 @@ print_help() {
   echo
   echo "Produce a local wheel and skip test and doc generation"
   echo "Example: ./make_wheel.sh --skip_test"
-  exit 1
+  return 1
```

Comment:
Will this change the behavior of the script? I think it may have been intended to early-exit when `print_help` is called.